lib/widgets/content.dart

```diff
@@ -266,44 +266,31 @@ class BlockContentList extends StatelessWidget {
   Widget build(BuildContext context) {
     return Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
       ...nodes.map((node) {
-        if (node is LineBreakNode) {
+        return switch (node) {
           // This goes in a Column.  So to get the effect of a newline,
           // just use an empty Text.
```

Comment:
Similarly on this comment, though in this case I think the expression form of `switch` works fine.